proto/v1alpha1/rode-attest.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package rode.v1alpha1;
+
+option go_package = "github.com/rode/rode/proto/v1alpha1";
+
+import "google/protobuf/timestamp.proto";
+
+message AttestPolicyRequest {
+  string policy = 1;
```

Comment:
should these all be required fields? I assume this message would be useless without a policy passed in

Reply:
Good call. I'll include that in the next PR.